packages/editor/src/hooks/default-autocompleters.js

```diff
@@ -16,7 +16,7 @@ import { userAutocompleter } from '../components';
 function setDefaultCompleters( completers = [] ) {
 	// Provide copies so filters may directly modify them.
 	completers.push( clone( userAutocompleter ) );
-
+	completers.push( clone( { ...userAutocompleter, triggerPrefix: '+' } ) );
```

Comment:
I believe this line should not be part of the PR, probably a leftover from testing for any regressions

Reply:
😱 That's embarrassing, thank you very very much.

Reply:
Don't worry, it's totally fine!